handlers/main.yml

```diff
@@ -16,3 +16,10 @@
     name: "gunicorn-{{ django_stack_active_gcorn_svc }}.service"
     state: started
     enabled: True
+
+- name: Update active svc remote fact
+  ini_file:
+    dest: /etc/ansible/facts.d/active_gunicorn_svc.fact
+    section: django
+    option: name
+    value: "{{ django_stack_active_gcorn_svc }}"
```

Comment:
Nit: the task name could be more descriptive here. Also, handlers typically have all lowercase names, but I don't see that enforced anywhere.

Consider forcing root:root and mode 0644 on this file, since its contents affect role execution.